resources/intro-reading.md

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Question: do we have a blurb at the top? or a link to this page somewhere? or do we want to just start this collection then add the other stuff later?

Reply:
yeah, I was thinking to start as a page that wasn't linked from anywhere, then when we feel like the content is good, we can link

agree will need some intro blurb, prolly will end up with groupings

Reply:
What do you think about adding links to non-reading resources (i.e. https://implicit.harvard.edu/implicit/takeatest.html)

Reply:
I like that idea! added issue to collect ideas/suggestions of what should be added before publishing https://github.com/bridgefoundry/bridgefoundry.github.io/issues/128